apps/polls/migrations/0002_auto_20170503_1524.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
```

Comment:
i'll leave the generated title this time but will name it manually next time. 
maybe we could also squash them when the feature is ready